system/jhtree/ctfile.cpp

```diff
@@ -1078,6 +1106,27 @@ CJHTreeBlobNode::~CJHTreeBlobNode()
 {
 }
 
+void CJHTreeBlobNode::unpack(const void *nodeData, bool needCopy)
+{
+    CJHTreeNode::unpack(nodeData, needCopy);
+    const byte *data = ((const byte *) nodeData) + sizeof(hdr);
+    char keyType = keyHdr->getKeyType();
+    if (keyType & HTREE_COMPRESSED_KEY)
+    {
+        expandedSize = keyHdr->getNodeSize();
+        keyBuf = expandKeys(data, expandedSize);
+    }
+    else
+    {
+        // Do blobs work on indexes that are not compressed - looking at old code I'm not sure they do
+        assertex(!(keyType & COL_PREFIX));
+        expandedSize = hdr.keyBytes + sizeof( __int64 );  // MORE - why is the +sizeof() there?
```

Comment:
I can't work it out either..  I suspect they don't work.  I think they are already compressed, but from memory the write will not correctly close the lzwcomp - so I think the blobs will be invalid.
Should probably unconditionally assume compressed and fix the writer to close.

Reply:
will be fixed in a later PR...